woocommerce/woocommerce-ios
# [WOO POS] update and use calculated totals

#12983 #13021


## Description


This PR is based on pfoUAQ-Iw-p2 with some addons and updates that accommodate changes on trunk. Combined work of @jaclync and @bozidarsevo 

- Totals calculation (order creation/update) is done when checkout button is tapped or items are changed while in the checkout phase.

## Next Steps

- Update the order for the payment process
- Update the totals view design according to final design



## Testing information


- Enter POS
- Add few items in the cart
- Tap "Checkout"
- Totals calculation should start (tax and total part are shimmering)
- When calculation is done, the correct amounts should show

## Screenshots


![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) 17 2 - 2024-06-17 at 18 38 17](https://github.com/woocommerce/woocommerce-ios/assets/6242034/c72db7cb-c763-42ba-bf27-cc8091f7747c)

---
